File: demo/cfd/cfd.f90
program main 
    use mpi
    use cpl, only : cfd_realm, CPL_create_comm
    implicit none

    integer :: rank, ierror, CFD_COMM

    call MPI_Init(ierror)
    call CPL_create_comm(cfd_realm, CFD_COMM, ierror)
    call MPI_Finalize(ierror)

end program main 

File: demo/cfd/cfd.cpp
#include "mpi.h"
#include "CPL.h"

int main (int argc, char** argv)
{
    MPI_Init (&argc, &argv);

    MPI_Comm realmComm;
    CPL::create_comm (CPL::cfd_realm, realmComm);

    MPI_Finalize();
}
